flow/record/jsonpacker.py

```diff
@@ -79,6 +79,12 @@ def pack_obj(self, obj: Any) -> dict | str:
                 "executable": obj.executable,
                 "args": obj.args,
             }
+        if isinstance(obj, fieldtypes.net.ipinterface):
+            return {
+                "ip": str(obj.ip),
+                "subnetmask": str(obj.netmask),
+                "network": str(obj.network),
+            }
```

Comment:
is returning a whole dictionary necessary? i would prefer to keep it simple and just use the string representation of the `ipinterface` object, eg: `192.168.1.10/24`.